android/app/build.gradle

```diff
@@ -76,7 +76,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "storybook/native",
+    entryFile: "storybook/native/index.js",
```

Comment:
Mind renaming it to `main.js` now? So it's similar to what storybook on web expects

Reply:
Yeah cool